src/cpu/CPUCore.cc

```diff
@@ -518,14 +519,32 @@ static constexpr void toHex(byte x, std::span<char, 3> buf)
 	buf[1] = toHex(x & 15);
 }
 
+template<typename T> void CPUCore<T>::disasmBlobCommand(
+	Interpreter& interp, std::span<const TclObject> tokens, TclObject& result) const
+{
+	if (tokens.size() < 4) {
+		interp.wrongNumArgs(2, tokens, "value addr");
+	}
+	std::string dasmOutput;
+	auto bin = tokens[2].getBinary();
+	unsigned len = instructionLength(bin);
+	if (len == 0 || len > bin.size()) {
+		throw CommandException("Wrong binary string size");
```

Comment:
Detail: "wrong size" is technically correct. But maybe a more helpful error message is "too small binary size" or "binary does not contain a complete instruction", or ...  (giving good error messages is hard).